### ISSUE TITLE
Improve the entering of coordinates in layer dimensions

### ISSUE THREAD

**Issue by riverole** [paperize/paperize#12]
**Is your feature request related to a problem? Please describe.**
It is a bit clunky the edition of coordinates/percentages in the layer edition. I find myself wanting to delete the numbers, and then start typing, and by that moment there's already a 0.0 which means that my new numbers, which were supposed to be for example 23.0, end up being 0.023 or such.

**Describe the solution you'd like**
I don't exactly know what would be a good solution, but it needs to be not so aggressive in putting the 0.0, since that creates friction from the end user.

**Comment by lorennorman** [comment]
Wooo this one's tough.

The user can type anything, but only certain values are valid. When the input gets sent to Vue/Vuex, it gets massaged into something valid, and that in turn updates the UI. The only lever there is the debounce time (how long it waits before applying the input.)

You can play with that value here (currently half a second):
https://github.com/paperize/paperize/blob/d2b9d14470efced3310283d151fc827b742e7e69/lib/store/util/vuex_resource.js#L159

The problem with extending that time is that it also extends the time before the template will re-render, and we value that being as snappy as possible.

Aside, the way I use these inputs is with my scroll wheel, kind of "throwing" the values around. When I change them, I select what is there and type over it (I don't try to zero it out first.) My method works, but obviously we can't ask all users to use it that way!

**Comment by Itangalo** [comment]
Yep, I've experienced this too. I eventually worked out that enter mode is "insert" in the text boxes. Making that clear to the user could go a long way, in my opinion. (People using Paperize are probably pretty used to adapt behavior to fit the tool.)
